core/frontend/src/components/vehiclesetup/MotorDetection.vue

```diff
@@ -0,0 +1,233 @@
+<template>
+  <v-dialog
+    v-if="is_sub"
+    v-model="dialog"
+    max-width="1000px"
+    @input="status = statuses.IDLE"
+  >
+    <template #activator="{ on, attrs }">
+      <v-btn
```

Comment:
The button should be disable if the vehicle is not in manual mode

Reply:
the button does put the vehicle in manual mode, same as in qgc

Reply:
So, we should update the following message:

"The vehicle should be in calm water, and the autopilot must be in manual mode to be able to run the test."

Since we are going to put in manual mode, it does not metter the mode that it is.

Reply:
removed